README.md

```diff
@@ -1,281 +1,133 @@
+* CloudFoundry
+* Build environments (i.e. docker containers, bosh releases etc)
+
+The full bootstrap process below is only needed the first time an environment is deployed.
+
+These instructions contain placeholders where the exact command may vary. The below table explains the purpose of those placeholders
+
+| Placeholder   | Purpose                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------|
+| `$ACCOUNT`    | The AWS account being targeted (e.g. `dev`, `staging`)|
+| `$ENV` | <p>The name of the environment being targeted. In the case of short lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars).</p> <p>Do not use the same `DEPLOY_ENV` value for both types of bootstraps, because that will cause resource allocation conflicts and things will break</p>|
+
+### Pre-requisites
+
+- [ ] Make
+- [ ] [GDS CLI](https://github.com/alphagov/gds-cli)
+- [ ] [jq](https://github.com/stedolan/jq)
+- [ ] Access to `paas-credentials` (private repository) and tools installed
+- [ ] Connection to the GDS VPN
+- [ ] Permission to assume the `Admin` role of the relevant AWS account (dev, ci, staging, production)
+- [ ] `AWS_DEFAULT_REGION` environment set the desired region for the environment
+- [ ] [Github OAuth application credentials](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+  If you're deploying a production/staging/shared dev environment (eg dev01, dev02), the app should belong to the alphagov organisation, and the credentials should be added to `paas-credentials` <br />
+  If you're deploying a short-lived environment (e.g. for penetration testing), they can be associated with your Github account. You can put them in your personal password store, and set the path to it using the `GITHUB_PASSWORD_STORE_DIR` environment variable
+
+## Deploying the bootstrap for CloudFoundry
+
+1. Clone `paas-bootstrap` at the tip of the `main` branch
+2. Launch Concourse Lite.
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV deployer-concourse bootstrap
+    ```
+3. Log in to Concourse List at http://127.0.0.1:8080/login using the credentials shown in the output from step 2
+4. Run the `create-bosh-concourse` pipeline
+5. When the pipeline reaches the `check-for-secrets` job, it will pause for up to an hour (checking every 10 seconds) to allow secrets to be uploaded:
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV upload-all-secrets
+    ```
+
+   **N.B** if you're deploying a short-lived environment (e.g. for penetration testing), the Github OAuth credentials will need to come from your personal `pass` store, and the path to it will be given by the `GITHUB_PASSWORD_STORE_DIR` env var
```

Comment:
```suggestion
   Note: if you're deploying a short-lived environment (for example, for penetration testing), the GitHub OAuth credentials will need to come from your personal `pass` store, and the path to it will be given by the `GITHUB_PASSWORD_STORE_DIR` env var
```